### ISSUE TITLE
Assets missing scenes?

### ISSUE THREAD

**Issue by bmarvo** [denetii/io_thps_scene#12]
Maybe they moved but should the assets folder contain a scenes folder? I noticed the previous versions with thug_tools does have this folder but those blends don't seem to work well in 2.8x.

**Comment by denetii** [comment]
Yes, these are currently missing as I'm undecided about whether to simply use migrated versions of the old .blend scenes, or take advantage of the game asset integration to directly import from the user's scene files. Will sort this out soon enough either way!
